worksheet_B/Mandelbrot set working.cpp

```diff
@@ -0,0 +1,97 @@
+// ConsoleApplication3.cpp : Defines the entry point for the console application.
+//
+
+#include "stdafx.h"
+
+float ImageWidth = 800.0;
+float ImageHeight = 800.0;
+int Iteration = 0;
+int maxIteration = 200;
+int Red = 0;
+int Green = 0;
+int Blue = 0;
+int minValue = 0;
+int maxValue = 255;

```

Comment:
Do all of these need to be global variables? Some would probably be better declared inside the `main` function.